libkbfs/folder_branch_ops.go

```diff
@@ -46,6 +46,9 @@ const (
 
 // Constants used in this file.  TODO: Make these configurable?
 const (
+	// The maximum block size in bytes
```

Comment:
Since this is exported, it's probably good practice to use a standard `// <VarName> does XYZ`-style comment. I'm surprised govet didn't catch this actually.